### PR TITLE
make config loadable, add server run and stop fns, dockerfile

### PR DESCRIPTION
**Pull request by tanmaykm**
- configuration is loadable from a configuration file
- service can be started with `Registrator.RegServer.main()` method
- docker build script
- handle multiple registries if configured (#26)

### CODE REVIEW

File: Manifest.toml [hunk 1]
```diff
@@ -34,16 +34,16 @@ deps = ["Random", "Serialization", "Sockets"]
 uuid = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 
 [[GitHub]]
-deps = ["Base64", "Compat", "Dates", "HTTP", "JSON", "MbedTLS", "Test"]
-git-tree-sha1 = "b988ce46c9b682acfeed4801cbaafa1ef954ad31"
+deps = ["Base64", "Compat", "Dates", "HTTP", "JSON", "MbedTLS", "Sockets", "Test"]
+path = "/home/tan/.julia/dev/GitHub"
```
Comment (StefanKarpinski) [inline Manifest.toml]: Did you mean to put a local absolute path in here?

Comment (tanmaykm) [reply Manifest.toml]: No. I was waiting for GitHub.jl to be tagged. I should have marked this as WIP. My bad.

Comment (StefanKarpinski) [reply Manifest.toml]: No worries, just thought I'd check!

**Comment by tanmaykm** [comment]
I think this is good to merge now

**Pull request merged**